docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+---
+title: "Uninstalling Knative"
+weight: 85
+type: "docs"
+---
+
+To uninstall an Operator-based Knative installation, follow the [Uninstall an Operator-based Knative Installation](#uninstall-an-operator-based-knative-installation) procedure below.
+To uninstall a YAML-based Knative installation, follow the [Uninstall a YAML-based Knative Installation](#uninstall-a-yaml-based-knative-installation) procedure below.
+
+
+## Uninstalling a YAML-based Knative installation
+
+To uninstall a YAML-based Knative installation:
+
+
+### Uninstalling optional Serving extensions
+
+Uninstall any Serving extensions you have installed by following the relevant steps below:
+
+{{< tabs name="serving_extensions" >}}
+
+{{% tab name="HPA autoscaling" %}}
+
+Knative also supports the use of the Kubernetes Horizontal Pod Autoscaler (HPA) for driving
+autoscaling decisions.
+The following command will install the components needed to support HPA-class autoscaling:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-hpa.yaml" >}}
+```
+
+<!-- TODO(https://github.com/knative/docs/issues/2152): Link to a more in-depth guide on HPA-class autoscaling -->
+
+{{< /tab >}}
+
+{{% tab name="TLS with cert-manager" %}}
+
+1. Uninstall the component that integrates Knative with cert-manager:
+
+   ```bash
+   kubectl delete -f {{< artifact repo="net-certmanager" file="release.yaml" >}}
+   ```
+
+1. Optional: if you no longer need cert-manager, uninstall it by following the steps in the
+[cert-manager documentation](https://cert-manager.io/docs/installation/uninstall/kubernetes/).
+
+   {{< /tab >}}
+
+{{% tab name="TLS via HTTP01" %}}
+
+Uninstall the `net-http01` controller by running:
+
+```bash
+kubectl delete -f {{< artifact repo="net-http01" file="release.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="TLS wildcard support" %}}
+
+Uninstall the components needed to provision wildcard certificates in each namespace by running:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-nscert.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="DomainMapping CRD" %}}
+
+To uninstall the `DomainMapping` CRD run:
```

Comment:
We should probably say "component" here.

Reply:
Thanks, I'll change that.